README.md

```diff
@@ -1,79 +1,19 @@
-# SOCIS Website
-
-This is the official website for the Society of Computing and Information Sciences (SOCIS) at the University of Guelph.
-
-This project was built using [React](https://reactjs.org/) and bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
-
-
-
-## How to run
-
-1. [First fork this repository](https://docs.github.com/en/github/getting-started-with-github/fork-a-repo)
-2. Clone this repository to your local machine
-3. Install dependencies using `yarn install`
-4. Run the server locally using `yarn start`. This will open the server on `localhost:8300` in your browser
-5. To run tests enter `yarn test`, this will lanch the test runner in interactive watch mode. Select `a` to run all tests
-6. To run linter enter `yarn lint`, to autofix linter errors, run `yarn lint-fix`
-
-## Build and run using docker
-
-1. Build: `docker build -t socis-web .`
-2. run: `docker run -p 8080:80 socis-web:latest`
-3. go too the url http://localhost:8080 and see the amazing version of the site.
-
-## Available Scripts
-
-In the project directory, you can run:
-
-`yarn start`
-
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
-
-The page will reload if you make edits.\
-You will also see any lint errors in the console.
-
-`yarn test`
+![CI testing](https://github.com/UoGSOCIS/socis_website/workflows/CI/badge.svg)
```

Comment:
I still think we should add some links here to the docs stuff. looking at a git repo people will normally just look at the readme and not go searching for things, I have never assumed that the `docs` folder would hold docs for me as a potential contributor, but rather I often assume that they are docs that go along with the project.

Reply:
To be fair does anyone read the documentation XD. But as an alternative, what if we put everything in the wiki instead? Might be the option that makes the most sense, and include very basic instructions on how to run in the README?

Reply:
I agree wiki might be good for some of that, as long as there is at least a link to it from the readme, basic instructions in readme + more detail in wiki  / docs works too (that is my preferred method). I really like readme's that include a quickstart then detailed usage is linked elsewhere